FiveCalls/FiveCalls/CallScriptViewController.swift

```diff
@@ -189,7 +190,27 @@ class CallScriptViewController : UIViewController, IssueShareable {
         present(alertController, animated: true, completion: nil)
     }
 
-    
+    func checkForNotifications() {
+        let permissions = OneSignal.getPermissionSubscriptionState()
+
+        if permissions?.permissionStatus.hasPrompted == .some(false) {
+            let alert = UIAlertController(title: R.string.localizable.notificationTitle(), message: R.string.localizable.notificationAsk(), preferredStyle: .alert)
+            alert.addAction(UIAlertAction(title: R.string.localizable.notificationAll(), style: .default, handler: { (action) in
+                OneSignal.promptForPushNotifications(userResponse: { (success) in
+                    OneSignal.sendTag("all", value: "true")
+                })
+            }))
+            alert.addAction(UIAlertAction(title: R.string.localizable.notificationImportant(), style: .default, handler: { (action) in
+                OneSignal.promptForPushNotifications(userResponse: { (success) in
+                    //
```

Comment:
Are you going to tag with `"important"` here?  What's the thinking?

Reply:
Having not used one signal segments / tags before, I was going to use no tags as an indication of only important messages, and the "all" tag for all messages. There's no level of push less than "just important messages" so it feels like a good baseline to me.